targets/targets.json

```diff
@@ -334,12 +333,26 @@
             "SLEEP",
             "SPI",
             "SPISLAVE",
-            "STDIO_MESSAGES"
-        ],
-        "default_lib": "small",
-        "release_versions": [
-            "2"
+            "STDIO_MESSAGES",
+            "USTICKER"
         ],
+        "overrides": {
+            "tickless-from-us-ticker": true,
+            "boot-stack-size": "0x300"
+        },
+        "supported_c_libs": {
+            "arm": [
+                "std", "small"
+            ],
+            "gcc_arm": [
+                "std", "small"
+            ],
+            "iar": [
+                "std"
+            ]
+        },
+        "c_lib": "small",
+        "supported_application_modes": ["bare-metal"],
```

Comment:
Migration guide was updated to use `supported_application_profiles` instead of `supported_application_modes`. Please note that this is in preparation for the functionality to be added in the build tool, it does not currently do anything.

Reply:
You still haven’t modified this.

Reply:
Thanks.  I fixed this.